core/appmodule/v2/comet.go

```diff
@@ -0,0 +1,11 @@
+package appmodule
+
+import "context"
+
+// HasPreBlocker is the extension interface that modules should implement to run
```

Comment:
This can be then moved in `module.go`. Additionally we need to update baseapp.

Reply:
this is a duplicate, baseapp needs the return while the new design uses consensus messages and querying directly from the module

Reply:
I think in our case, consistency and limiting conflicting apis in better.
Additionally, we cannot have x/upgrade compatible with baseapp and server/v2 if we don't modify in baseapp.
In baseapp, we can update it every pre block instead of checking the returned value.

My point about moving this into core/appmodule/v2/module.go still stand btw

Reply:
gonna remove this and we go with there is no v2 yet